src/features/ping/dril.ts

```diff
@@ -15,6 +15,7 @@ import { Brain } from "../../brain"
 
 const NEVER = "https://twitter.com/dril/status/247222360309121024"
 const NO = "https://twitter.com/dril/status/922321981"
+const GEANS = "https://pbs.twimg.com/media/DmO9KAKUUAAfRUN?format=jpg"
```

Comment:
Nice. However, you should copy the image from that link and upload it to a Discord server, then use the link from Discord instead!